bash_loading_animations.sh

```diff
@@ -148,40 +89,22 @@ exit 0
 ################### show loading animations in your script. ###################
 ###############################################################################
 
+:<<'EXAMPLES'
+
+# load in the functions and animations
+source /path/to/bash_loading_animations.sh
+
 # Show a loading animation for the command "foo"
-start_loading_animation "${name_of_the_animation[@]}"
+BLA::start_loading_animation "${BLA_name_of_the_animation[@]}"
 foo
-stop_loading_animation
+BLA::stop_loading_animation
 
-# If the command prints some output in the terminal, you may want to add:
+# If the command prints some output in the terminal, you may want to add
+# one of the following:
 foo 1> /dev/null # hide standard output
 # or
 foo 2> /dev/null # hide error messages
 # or
 foo &> /dev/null # hide all output
 
-###################################### 5 ######################################
```

Comment:
By loading as a library, you no longer need to provide shorter cut 'n' paste snippets.